### ISSUE TITLE
Using multiple colors when multiple ICA components in the reconstructed latent source figure

### ISSUE THREAD

**Issue by crsegerie** [mne-tools/mne-python#9417]
![image](https://user-images.githubusercontent.com/38733898/119115926-02996e80-ba28-11eb-9d22-92a37ea2230f.png)

In the above figure, the colors look the same for me, and I don't know which component is which one.

**Comment by marsipu** [comment]
Does this come from something like:

```python
ecg_evoked = create_ecg_epochs(raw).average()
ica.plot_sources(ecg_evokeds)
```
And it probably should only apply for evokeds?
I could look into it in the next time if nobody else does it yet

**Comment by hoechenberger** [comment]
> Does this come from something like:
> 
> ```python
> ecg_evoked = create_ecg_epochs(raw).average()
> ica.plot_sources(ecg_evokeds)
> ```

Yep!

> And it probably should only apply for evokeds?

I suppose so, yes!

**Comment by marsipu** [comment]
Would you want different colors just for the excluded components or for every component?

**Comment by hoechenberger** [comment]
Only for the excludes, they are already labeled and appear in the legend

**Comment by drammock** [comment]
closed by #9444
